constantine/eth_verkle_ipa/eth_verkle_constants.nim

```diff
@@ -43,22 +43,16 @@ type
     D*: EC_P
 
 const
-  VerkleDomain*: int = 256
+  EthVerkleDomain*: int = 256
 
 type VerkleIPAProofSerialized* = array[544, byte]
 
 type VerkleMultiproofSerialized* = array[576, byte]
 
-type
-  PrecomputedWeights* = object
-    barycentricWeights*: array[512,Fr[Banderwagon]]
-    invertedDomain*: array[510,Fr[Banderwagon]]
-
 type
   IPASettings* = object
-    SRS*: array[VerkleDomain,EC_P]
-    Q_val*: EC_P
```

Comment:
We can inline the generator at compile-time with Nim. And store it in affine coordinates instead of projective to reduce memory cost.